_posts/2017-01-13-github_changelog_generator-usage.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Using github_changelog_generator
+date: 2017-01-13
+github_username: rnelson0
+twitter_username: rnelson0
+---
+
+With the upgrade to Puppet 4 in all of our modules, we have also added the gem [github_changelog_generator](https://github.com/skywinder/github-changelog-generator). This is an automated tool to generate a `CHANGELOG.md` based on the Issues and PRs from a github repo with minimal manual intervention. No more forgetting about an important change or describing it wrong, the gem handles that for you.
+
+The changelog contents are based on a few things:
+* The current version of the Puppet module per `metadata.json`
```

Comment:
should we edit the version in `metadata.json` before or after the generator parsed it?